admin_manual/enterprise/user_management/saml_2.0_sso.rst

```diff
@@ -154,17 +154,42 @@ Testing
 Configuring  SSO
 ----------------
 
-- On the ADFS Server add ``Windows Authentication`` to the ``Service``->``Authentication Methods`` for ``Intranet``
+- On the ADFS Server:
+    - Add ``Windows Authentication`` to the ``Service``->``Authentication Methods`` for ``Intranet``
+    - Run the following Powershell script for Firefox:
+    
+``
+# Save the list of currently supported browser user-agents to a variable
+$browsers=Get-AdfsProperties | Select -ExpandProperty WIASupportedUseragents
+
+# Add Mozilla/5.0 user-agent to the list
+$browsers+="Mozilla/5.0"
+
+# Apply the new list
+Set-AdfsProperties -WIASupportedUseragents $browsers
+
+# Turn off Extended Protection
+#Set-ADFSProperties –ExtendedProtectionTokenCheck None
+
+# Restart the AD FS service
+Restart-Service adfssrv
+``
 - On the windows client:
-  In the ``Internet Settings``->``Security``->``Local Intranet``
-  + Click on "Sites"
-  + Click on "Advanced"
-  + Add your adfs machine with `https://<adfs server fqdn>/` and click ok.
-  + Click on `customize level`
-  + Find `User Authtification`
-  + Check `Automatic login only for Intranet zone`
-
-Now if you logged into the domain and open your ownCloud server in Internet Explorer or Edge with URI "oc-shib" you should get directly to your ownCloud files without a login.
+  - For Internet Explorer / Edge / Chrome
```

Comment:
s/"For Internet Explorer / Edge / Chrome"/"For Internet Explorer, Edge, and Chrome"/g